Source/Core/DolphinWX/Frame.h

```diff
@@ -109,6 +109,7 @@ class CFrame : public CRenderFrame
   bool RendererHasFocus();
   bool RendererIsFullscreen();
   void OpenGeneralConfiguration(wxWindowID tab_id = wxID_ANY);
+  void CycleProfile(bool forward);
```

Comment:
This doesn't need to be public. This should also likely use an enum class instead of a boolean to signify what behavior it's doing.

Reply:
Fixed.